guide/source/2.8-migration.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Migrating to Meteor 2.8
+description: How to migrate your application to Meteor 2.8.
+---
+
+Meteor `2.8` introduce the new MongoDB Package Async API. For a complete breakdown of the changes, please refer to the [changelog](http://docs.meteor.com/changelog.html).
+
+For this new async API, we have new methods like `findOneAsync`, which behaves exactly like the `findOne` method, but it now returns a promise that needs to be resolved to get the data.
+
+<h3 id="why-like-this">Why this is important?</h3>
+
+You may know that on Meteor we use a package called [Fibers](https://github.com/laverdet/node-fibers). This package is what makes possible to call async function, like `db.findOne()`, inside Meteor in a sync way (without having to wait for the promise to resolve).  
+
+But starting from Node 16, Fibers will stop working, so Meteor needs to move away from Fibers, otherwise, we'll be stuck on Node 14.
+
+If you want to know more about the plan, you can check this [discussion](https://github.com/meteor/meteor/discussions/11505).
+
+<h3 id="why-now">Why doing this now?</h3>
+
+This will be a considerable change for older Meteor applications, and some parts of the code of any Meteor app will have to be adjusted eventually.
+
+But we're going to do this in a way that has the most negligible impact possible on the applications and over time.
+
+With this version, you'll be able to start preparing your app for the future by replacing your current MongoDB methods with the new async ones.
+
+<h3 id="what-is-new">What's new?</h3>
+
+Here are the newly added methods (you can see this description and the code [here](https://github.com/meteor/meteor/pull/12028)):
+
+**Added async methods on collections.**
+ - All async methods have an Async suffix to their names. These are: `createCappedCollectionAsync`, `createIndexAsync`, `dropCollectionAsync`, `dropIndexAsync`, `findOneAsync`, `insertAsync`, `removeAsync`, `updateAsync`, and `upsertAsync`.
+
+**Added async methods on cursors.**
+ - All async methods have an Async suffix to their names. These are: `countAsync`, `fetchAsync`, `forEachAsync`, and `mapAsync`.
+ - There's also `[Symbol.asyncIterator]`, so this code should work:
+    ```js
+    for await (const document of collection.find(query, options)) /* ... */
+    ```
+**A few internal methods are now async.**
+ 
+ - These are: `MongoConnection` (constructor), `MongoInternals.RemoteCollectionDriver` (constructor), `MongoInternals.defaultRemoteCollectionDriver`, `OplogHandle` (constructor).
+
+Adding these async counterparts allows gradual migration, as both versions will coexist for the time being. At the moment, all async methods are calling sync one directly, so all hooks and wrappers should work as expected.
```

Comment:
Since https://github.com/meteor/meteor/pull/12097 they no longer are. (It also impacts the comments below.)